docs/src/tool/index.md

```diff
@@ -105,18 +104,24 @@ named `default`.
 + The environment variable `REACH_DEBUG`, if set to any non-empty value, enables debug messages from the Reach compiler, which will appear in the console.
   This debug information includes: the estimated cost of the contract on Algorand.
   This variable automatically enabled `--intermediate-files`.
+  
+A `reach compile` usage example is available in [Overview](##over-compile).
 
-## {#ref-usage-init} `reach init`
+# {#ref-usage-init} `reach init`
```

Comment:
Weren't these changes made before? I think you need to rebase

Reply:
You should rebase this and I believe it will get fixed

Reply:
I tried to rebase to master, but it is giving me the invalid paths from before, I tried to do the --reapply-cherry-picks, and it didn't work either.